lib/Config/Config.php

```diff
@@ -44,13 +44,18 @@ public function __construct() {
 	 * Get the merged data from all the input sources provided in the constructor.
 	 *
 	 * @since  0.0.0
-	 * @param  string $purpose Name of the purpose.
-	 * @return array           Array of Projects.
+	 * @param  string   $purpose         Name of the purpose.
+	 * @param  string[] $wanted_projects Array of project names to get configuration for. Default is
+	 *                                   an empty array, all projects are retrieved.
+	 * @return array                     Array of Projects.
 	 */
-	public function get( string $purpose ) : array {
+	public function get( string $purpose, array $wanted_projects = [] ) : array {
 		$project_configs = [];
 		foreach ( $this->config as $project_name => $config ) {
-			if ( ! isset( $config[ $purpose ] ) ) {
+			if (
+				! isset( $config[ $purpose ] )
+				|| ( ! empty( $wanted_projects ) && ! in_array( $project_name, $wanted_projects, true ) )
```

Comment:
This should be enough.
```suggestion
				|| ! in_array( $project_name, $wanted_projects, true )
```

Reply:
We need the `! empty` in order to get all the projects when `$wanted_projects` is empty. With just `! in_array`, we wouldn't get any projects.